frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -74,14 +86,39 @@ export const VirtualMachinesDetailsPage: React.FC<VirtualMachinesDetailsPageProp
     disksPage,
   ];
 
-  return (
-    <DetailsPage {...props} menuActions={menuActionsCreator} pages={pages} resources={resources} />
+  const vmLoaded = getLoadedData(vm);
+  const vmiLoaded = getLoadedData(vmi);
+
+  return vmLoaded ? (
+    <DetailsPage
+      {...props}
+      kind={VirtualMachineModel.kind}
+      kindObj={VirtualMachineModel}
+      menuActions={menuActionsCreator}
+      pages={pages}
+      resources={resources}
+      breadcrumbsFor={breadcrumbsForVMPage(props.match)}
+    />
+  ) : (
+    vmiLoaded && (
```

Comment:
we should take into account that the data should not be editable in the VMI like boot order/nics etc.

Reply:
:+1: disabled edit kebabs on nics and discs